File: python/17c_200B_Drinks.py
def main():
    n = int(input())
    per = 0.0
    drinks = input().split()
    for i in drinks:
        per += int(i)
    per = (per / (n * 100.0)) * 100.0
    print(f"{per:.7f}")

if __name__ == "__main__":
    main()

File: cpp/17c_200B_Drinks.cpp
#include<bits/stdc++.h>

int main(){
    int n;
    std::vector<float> drinks;
    float per, temp;
    std::cin>>n;
    for(int i = 0; i < n; i++){
        std::cin>>temp;
        drinks.push_back(temp);
        per += temp;
    }
    per = (per / (n * 100.0)) * 100.0;
    std::cout<<per;
    return 0;
}